src/lib/CLValue/AccountHash.ts

```diff
@@ -12,19 +9,16 @@ import {
   ToBytesResult,
   ACCOUNT_HASH_LENGTH,
   resultHelper,
-  ACCOUNT_HASH_ID
+  ACCOUNT_HASH_ID,
+  CLByteArrayType
 } from './index';
 
-export class CLAccountHashType extends CLType {
-  linksTo = CLAccountHash;
-  tag = -1;
-
-  toString(): string {
-    return ACCOUNT_HASH_ID;
-  }
+// AccountHash is an alias, not a fully functional CLType so uses the same CLTypeTag as ByteArray
+export class CLAccountHashType extends CLByteArrayType {
+  linksTo = ACCOUNT_HASH_ID;
 
-  toJSON(): string {
-    return this.toString();
+  constructor() {
+    super(32);
```

Comment:
```suggestion
    const size = 32;
    super(size);
```

Reply:
This 32 looks like CL_BYTE_ARRAY_MAX_LENGTH, if this can be whether imported here maybe, or have it in one place defined maybe.

Reply:
replaced for `ACCOUNT_HASH_LENGTH` that was defined in `constants`